browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -2082,6 +2082,15 @@ index 848da780c8b2c71e50542a66afddf5973a279c6c..642e715265971685257fffefc288ca97
  
  # FIXME: This is on by default in WebKit2. Perhaps we should consider turning it on for WebKitLegacy as well.
  MediaCapabilitiesExtensionsEnabled:
+@@ -875,7 +875,7 @@ MediaSessionPlaylistEnabled:
+   type: bool
+   humanReadableName: "Experimental MediaSession playlist API"
+   humanReadableDescription: "Enable experimental MediaSession playlist API"
+-  condition: ENABLE(MEDIA_SESSION_PLAYLIST)
++  condition: ENABLE(MEDIA_SESSION_COORDINATOR) && ENABLE(MEDIA_SESSION_PLAYLIST)
```

Comment:
Can we revert this part of the change or are you planning to upstream it? We anyway disable MEDIA_SESSION_PLAYLIST so this condition is already false in playwright.